app/(landing)/views/social-proof/social-proof-stat-item.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { interFont } from '../../../shared/fonts/Inter'
+import styles from './social-proof.module.scss'
+
+interface StatItemProps {
+  number: string
+  description: string
+  children?: React.ReactNode
+}
+
+const SocialProofStatItem: React.FC<StatItemProps> = ({ number, description, children }) => {
+  return (
+    <div className={styles.statItem}>
+      <div className={`${styles.number} ${interFont.className}`}>{number}</div>
+      <div className={styles.description}>{description}</div>
+      {children && <div className={styles.children}>{children}</div>}
+    </div>
+  )
+}
+
```

Comment:
In general, this is good, but in future you can implement it through nesting
```suggestion

/**
* @example
* <SocialProofStatItem>
*   <ProofStatNumber>20 000+</ProofStatNumber>
*   <ProofStatNumber>Members in community</ProofStatNumber>
*   <SocialProofLinks />
* </SocialProofStatItem>
*/
const SocialProofStatItem: React.FC<React.PropsWithChildren> = ({ children  }) => {
  return (
    <div className={styles.statItem}>
    {children}
    </div>
  )
}

export const ProofStatNumber: React.FC<React.PropsWithChildren> = ({children}) =>
 <div className={`${styles.number} ${interFont.className}`}>{children}</div>

export const ProofStatDescription: React.FC<React.PropsWithChildren> = ({children}) =>
 <div className={styles.description}>{children}</div>

```

Reply:
Got it, might be at next update